base/base.jl

```diff
@@ -30,21 +30,17 @@ end
 The parameters to a function call do not match a valid signature. Argument `msg` is a
 descriptive error string.
 """
-type ArgumentError <: Exception
+mutable struct ArgumentError <: Exception
     msg::AbstractString
 end
 
-#type UnboundError <: Exception
```

Comment:
I don't think it was ever used. Surprising how long those 3 lines managed to hang around.